services/drupal/web/modules/custom/epa_migrations/src/Plugin/migrate/source/EpaMultivalueField.php

```diff
@@ -35,6 +35,9 @@ public function query() {
       $query = $this->select($field_table, 'fd')
         ->fields('fd');
 
+      // Add an expression to let us generate a unique field for use as high water mark.
+      $query->addExpression('CONCAT(fd.revision_id, "-", fd.delta)', 'unique_id');
```

Comment:
SqlBase uses `$conditions->condition($high_water_field, $high_water, '>')` and MySQL doesn't allow comparison with a computed column: `Unknown column 'unique_id' in 'where clause'`. So I think our plugin will have to override initializeIterator() and use `> CONCAT(...`.